### PR TITLE
create page-part

### PR DESCRIPTION
**Pull request by Nazar0n**

### CODE REVIEW

File: styles/content.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    max-width: 1400px;
+    background-color: #E8E8E8;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.header-first {
+    display: flex;
+    font-family: 'Work Sans';
```
Comment (Nick-Shcherban) [inline styles/content.css]: do you really need to connect font again ? (has already been connected in head)

File: styles/content.css [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+.description_content {
+    display: flex;
+    font-family: 'Work Sans';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 24px;
+    line-height: 34px;
+    text-align: center;
+    text-align: center;
```
Comment (Nick-Shcherban) [inline styles/content.css]: doubled

File: styles/content.css [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+@media (min-width: 600px) {
+    .page-buttons {
+        width: 160px;
+        height: 48px;
+    }
+}
+
+.blue {
+    background-color: #3040C4;
+    color: white;
```
Comment (Nick-Shcherban) [inline styles/content.css:114]: better to use rgb code everywhere

**Review by Nick-Shcherban — APPROVED** [review]
minor comments

File: index.html [hunk 4]
```diff
@@ -45,6 +46,47 @@ <h1 class="title_hiring"> We're hiring </h1>
     </div>
     <!-- здесь заканчивается Колина часть html -->
 
+    <!--Nazar part-->
+
+    <div class="page-content">
+        <div class="header-first"><span>Engage Visitors</span></div>
```
Comment (velmyk) [inline index.html]: Do you need a `span` here?

Comment (velmyk) [reply index.html]: Or maybe `div` is not needed. No need to add an additional tag if thay do nothing

File: index.html [hunk 5]
```diff
@@ -45,6 +46,47 @@ <h1 class="title_hiring"> We're hiring </h1>
...
+        <div class="header-second"><span>Understand your audience</span></div>
+        <div class="description_content">
```
Comment (velmyk) [inline index.html:54]:
```
    <p class="description_content">
            Early stage company? Eligible applicants get all of our products for
            just $49 a month.
 </p>
```

File: index.html [hunk 6]
```diff
@@ -45,6 +46,47 @@ <h1 class="title_hiring"> We're hiring </h1>
...
+          <p>
+            Early stage company? Eligible applicants get all of our products for
+            just $49 a month.
+          </p>
+        </div>
+        <div class="buttons">
+          <input class="blue page-buttons" type="button" value="Get Started" />
```
Comment (velmyk) [inline index.html]: Use `button` tag instead of `<input type="button"...`

File: index.html [hunk 7]
```diff
@@ -45,6 +46,47 @@ <h1 class="title_hiring"> We're hiring </h1>
...
+        </div>
+        <div class="pics">
+          <div class="pictures">
+            <img class="pic" src="images/developer.png" alt="Developer" />
+            <br />
+            <span class="pic-text">Developers</span>
+          </div>
+          <div class="pictures">
+            <img class="pic" src="images/tv.png" alt="TV" />
+            <br />
```
Comment (velmyk) [inline index.html]: Don't use `br` tag
`.pictures` can be `display: flex;` and `flex-direction: column`

File: index.html (hunk 4 above)
Comment (Nazar0n) [reply index.html]: Sorry i experimented with `span's` & `div's` and forgot delete. Was fixed

**Review by velmyk — APPROVED** [review]

**Pull request merged**